### PR TITLE
DIV-6325: clean up

### PR DESCRIPTION
**Pull request by yarpo**
Making sure we re-use what can be re-used.

https://tools.hmcts.net/jira/browse/DIV-6325

### CODE REVIEW

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    "PageFieldDisplayOrder": 6,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "tempEditDerivedAddresses",
+    "PageLabel": "Temporary - Edit derived addresses",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
```
Comment (jvictorcf) [inline definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json:81]: These should not be in production.

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    "PageFieldDisplayOrder": 7,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "tempEditDerivedAddresses",
+    "PageLabel": "Temporary - Edit derived addresses",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```
Comment (jvictorcf) [inline definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json:94]: Are you sure these labels should be in prod? It looks like a strange thing in a clean-up PR.

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json (hunk 1 above)
Comment (yarpo) [reply definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json:81]: yes, well spotted :)

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json (hunk 2 above)
Comment (yarpo) [reply definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json:94]: no, this file should be 

```
[]
```

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json [hunk 3]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (Dizrupto) [inline definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json:1]: Do we need those files? Not used yet

**Pull request merged**